src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Shared/DraftApprenticeship.cshtml

```diff
@@ -34,375 +32,326 @@
-                        <input asp-for="ActualStartYear" min="1900" max="9999" class="govuk-input govuk-date-input__input govuk-input--width-4" type="number">
+                        <label asp-for="BirthYear" class="govuk-label govuk-date-input__label">Year</label>
+                        <input asp-for="BirthYear" min="1900" max="9999" class="govuk-input govuk-date-input__input govuk-input--width-4" type="number">
                     </div>
                 </div>
             </div>
         </fieldset>
     </div>
+}
+
+@if (Model is EditDraftApprenticeshipViewModel)
+{
+    <h1 class="govuk-heading-xl">Edit training details</h1>
+}
+else
+{
+    <h1 class="govuk-heading-xl">Add training details</h1>
+}
+
+<div class="govuk-form-group">
+    <p class="govuk-!-font-weight-bold govuk-!-margin-bottom-0">Apprenticeship training course</p>
+    <p class="govuk-body">
+        @Model.CourseName
+        <button id="change-course-link" type="submit" name="ChangeCourse" value="Edit" class="das-button--inline-link">
+            @editCourseLabel<span class="govuk-visually-hidden"> apprenticeship training course</span>
+        </button>
+    </p>
+</div>
+<input type="hidden" asp-for="CourseCode" class="govuk-radios__input" value="@Model.CourseCode">
+<input type="hidden" asp-for="DeliveryModel" class="govuk-radios__input" value="@(Model.DeliveryModel ?? DeliveryModel.Regular)">
+
+@if (Model.HasMultipleDeliveryModelOptions || Model.HasUnavailableFlexiJobAgencyDeliveryModel || Model.HasChangedDeliveryModel)
+{
+    <div class="govuk-form-group">
+        <p class="govuk-!-font-weight-bold govuk-!-margin-bottom-0">Apprenticeship delivery model</p>
+        <p class="govuk-body">
+            @Model.DeliveryModel.ToDescription()
+            @if (Model.HasMultipleDeliveryModelOptions || Model.HasUnavailableFlexiJobAgencyDeliveryModel)
+            {
+                <button id="change-delivery-model-link" type="submit" name="ChangeDeliveryModel" value="Edit" class="das-button--inline-link">
+                    Edit<span class="govuk-visually-hidden"> delivery model</span>
+                </button>
+            }
+        </p>
+    </div>
+}
+else if (Model.DeliveryModel != DeliveryModel.Regular)
+{
+    <dl class="das-definition-list">
+        <dt class="das-definition-list__title">Apprenticeship delivery model</dt>
+        <dd class="das-definition-list__definition govuk-!-margin-bottom-4">@Model.DeliveryModel.ToDescription()</dd>
+    </dl>
+}
 
-    <div class="govuk-form-group  @Html.AddClassIfPropertyInError(x => x.EndDate, "govuk-form-group--error")">
+<div id="start-date-section" class="govuk-form-group  @Html.AddClassIfPropertyInError(x => x.StartDate, "govuk-form-group--error")">
+    <fieldset class="govuk-fieldset" aria-describedby="start-date-hint" role="group">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+            @startDateLabel
```

Comment:
Should both the "planned" and "actual" start date be visible simultaneously here? Again, just going off the mockups here, it looks like the "actual" start date is not visible for non-pilot providers (but "planned" is), and vice-versa for pilot providers.